insights/parsers/cpuinfo.py

```diff
@@ -119,7 +119,8 @@ def parse_content(self, content):
             "model": "model_ids",
             "cpu family": "families",
             "cpu MHz": "clockspeeds",
-            "cache size": "cache_sizes"
+            "cache size": "cache_sizes",
+            "flags": "flags"
```

Comment:
@kgrant-rh would you mind to add `flags` to the sample input/output documentation?